### DISCUSSION TITLE
Local QField Cloud mirror and QGIS workflow

### DISCUSSION THREAD

**Discussion by ntrip6** [opengisch/QField#2882] [Q&A]
I'm hoping for some clarification about the Local Mirror and continued working in a QGIS project. After the upload to QFieldCloud I then can choose between working in the [project name]_cloud.qgs (local mirror) or the original QGIS [project name].qgz project.

After returning from the field my changes appear in the _cloud.qgs project so I'm tempted to keep working with it.
However I see this alert in the documentation
**Attention:** This workflow does not support changing the GeoPackage on the desktop, as being file base, the whole GeoPackage will be replaced. This means that data can only be digitized using QFieldCloud.

Is it safe to keep working with the projectname_cloud.qgs version? 
I've looked through docs in particular https://docs.qfield.org/get-started/tutorials/get-started-qfc/
but I'm hoping for a recommended workflow.

**Comment by suricactus** [comment]
Can you give a little bit more details what provider you have for your layers and how you created your QFieldCloud project?

**Reply by ntrip6** [reply]
I have a few vector features in a gpkg (these are edited in the field), and for the read-only layers I have some local Lidar geotiffs and QuickMapServices imagery layers. In terms of creating the QFieldCloud project I just used "Create new project" and "Convert currently open project" in QGIS 3.22 (Windows 10).  The QField Cloud tool copies things to the windows path 
_C:\Users\[username]\QField\cloud\project\ and the new sync gpkg in that path

I assume the simple answer to my original question is that I should not work out of this \cloud\project folder back in QGIS after returning from fieldwork. I should sync from the cloud then transfer things back into my "real" GPKG where it started but I'm not clear on how these data are reintegrated back into the original GPKG file. I could use something like _Processing>Merge Vector Layers_ in QGIS but that seems unlikely to be the recommended workflow. 
 Is there some desktop synchronization step I'm missing? Thanks

**Reply by ntrip6** [reply]
The QFieldCloud Sync tool shows this dialog
![QFieldCloud](https://user-images.githubusercontent.com/480545/168374608-5ca0e7c5-38b8-4187-9627-a20423efb01b.JPG)

which will sync from the cloud to the Local Directory. This works well. However, how do I now get the original project QGZ and GPKG to sync with the Local Directory? It is referred to as a Local Mirror which I took to mean this directory would somehow "mirror" my actual working directory in the way that Disconnected Editing (ESRI GDB) can eventually be reunited and reconciled. In fact all the new work from QField is now only in the Cloud and the Local Mirror directory and my original project is now out of date. Perhaps I misunderstood the word "Mirror" because it only mirrors the local directory at creation time but are subsequently disconnected. Or did I set it up incorrectly?
